src/Menu/Address.java

```diff
@@ -0,0 +1,34 @@
+package Menu;
+
+public class Address {
+
+        private String roadName;
+        private Integer houseNum;
+        private String district;
+        private String city;
+        private String estate;
+
+		public Address(){
+
+        }
+
+        //addres cronstuctor
+        public Address(String roqdName, Integer houseNum, String bairro, String city, String estate){
```

Comment:
```suggestion
        public Address(String streetName, Integer number, String district, String city, String state){
```

Cuidado pra não misturar os idiomas no meio do código